contracts/daipay.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.2;
+
+/**
+ * @title ERC20Basic
+ * @dev Simpler version of ERC20 interface
+ * @dev see https://github.com/ethereum/EIPs/issues/179
+ */
+contract ERC20Basic {
+    function totalSupply() public view returns (uint256);
+    function balanceOf(address who) public view returns (uint256);
+    function transfer(address to, uint256 value) public returns (bool);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+}
+
+
+contract ERC20 is ERC20Basic {
```

Comment:
Is the ERC20Bsic functionality required? Are we intending to add permission, to allow a manager disburse the funds?

Reply:
i dont think a permission is necessary, considering we're calling the transferFrom function which will perform necessary checks. so as long as you're address has the required token you're good. but i agree with your comment on the ER20Basic it's redundant